cni/network/network.go

```diff
@@ -517,16 +521,23 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 		}()
 	}
 
+	msg = fmt.Sprintf("Allocated IPAddress from ipam:%+v v6:%+v", result, resultV6)
```

Comment:
Instead of individual calls to metrics and logs, can you probably combine them into a single call ?

Reply:
https://github.com/Azure/azure-container-networking/blob/da719d7a7b5fdf89cd841ecbb966c73bd3fc0ee1/npm/metrics/ai-utils.go#L51 

We did something similar in NPM,